pom.xml

```diff
@@ -3,40 +3,41 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>2.11</version>
+    <version>3.43</version>
+    <relativePath />
   </parent>
 
   <artifactId>form-element-path</artifactId>
   <version>1.9-SNAPSHOT</version>
   <packaging>hpi</packaging>
   <name>Form element path plugin</name>
   <description>Introduce unique path attributes for UI controls to facilitate Jenkins UI testing.</description>
-  <url>https://wiki.jenkins-ci.org/display/JENKINS/Form+Element+Path+Plugin</url>
+  <url>https://wiki.jenkins.io/display/JENKINS/Form+Element+Path+Plugin</url>
 
   <properties>
-    <jenkins.version>1.580</jenkins.version>
-    <java.level>6</java.level>
+    <jenkins.version>2.176.1</jenkins.version>
```

Comment:
Newer than [this](https://github.com/jenkinsci/jenkins/blob/3ba5ca46c3b8e3df79b0550ff17e652006beef05/core/src/main/resources/jenkins/split-plugins.txt#L30) and thus thought to avoid [this](https://ci.jenkins.io/job/Core/job/acceptance-test-harness/job/master/158/testReport/junit/core/InstallWizardTest/java_8_split7___wizardInstallCustomPluginsTest/), assuming [this](https://github.com/jenkinsci/acceptance-test-harness/blob/8cd0e2b67c84f33fe286d0f0bd53a2d8c986b490/src/main/java/org/jenkinsci/test/acceptance/FallbackConfig.java#L305) were updated.

Reply:
(working around [JENKINS-28942](https://issues.jenkins-ci.org/browse/JENKINS-28942))